### PR TITLE
Update german.xml to v8.4.7

### PR DESCRIPTION
**Pull request by schnurlos**
Fix sorting of some lines.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/german.xml [hunk 1]
```diff
@@ -1598,7 +1614,13 @@ Suche in allen Dateien, exkludiere alle Verzeichnisse log oder logs rekursiv:
 			<contextMenu-styleAlloccurrencesOfToken value="Alle Vorkommen hervorheben" />
 			<contextMenu-styleOneToken value="Dieses Vorkommen hervorheben" />
 			<contextMenu-clearStyle value="Alle Hervorhebungen entfernen" />
-			<contextMenu-PluginCommands value="Plugin-Befehle" />
```
Comment (donho) [inline PowerEditor/installer/nativeLang/german.xml:1601]: Restore this line

**Review by donho — CHANGES_REQUESTED** [review]
Please check the latest change: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12404

**Comment by schnurlos** [comment]
@donho Done

**Comment by schnurlos** [comment]
I'm not sure what changes are missing now. Otherwise I can delete this commit and create a new one.

**Comment by schnurlos** [comment]
Update to 8.4.8

**Pull request closed without merging**